rubygems-update.gemspec

```diff
@@ -6,14 +6,14 @@ Gem::Specification.new do |s|
   s.authors = ["Jim Weirich", "Chad Fowler", "Eric Hodel", "Luis Lavena", "Aaron Patterson", "Samuel Giddins", "André Arko", "Evan Phoenix", "Hiroshi SHIBATA"]
   s.email = ["", "", "drbrain@segment7.net", "luislavena@gmail.com", "aaron@tenderlovemaking.com", "segiddins@segiddins.me", "andre@arko.net", "evan@phx.io", "hsbt@ruby-lang.org"]
 
-  s.summary = "RubyGems is a package management framework for Ruby."
+  s.summary = "RubyGems is a package management framework for Ruby—this gem allows the gem command to update itself."
```

Comment:
I'd go with

```suggestion
  s.summary = "RubyGems is a package management framework for Ruby. This gem is downloaded and installed by `gem update --system`, so that the `gem` CLI can update itself."
```